.pre-commit-config.yaml

```diff
@@ -1,9 +1,17 @@
 repos:
-- repo: local
-  hooks:
-  - id: scan-secrets
-    name: Scan secrets
-    entry: ./scripts/githooks/scan-secrets.sh
-    args: ["check=staged-changes"]
-    language: script
-    pass_filenames: false
+  - repo: local
+    hooks:
+      - id: scan-secrets
+        name: Scan secrets
+        entry: ./scripts/githooks/scan-secrets.sh
+        args: ['check=staged-changes']
+        language: script
+        pass_filenames: false
+  - repo: https://github.com/astral-sh/ruff-pre-commit
```

Comment:
I think we should add this to the CI as well, it's useful in cases where somebody new contributes but hasn't got pre-commit working for whatever reason

Reply:
We're currently linting in CI with this change [here](https://github.com/NHSDigital/manage-breast-screening/pull/88/commits/8c2c13666a2b38d1ceb11635571576cff0927d37#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R52) . Was there something additional needed there?

Reply:
Ah no, that's it 👍🏻